lib/routes/sorrycc/index.ts

```diff
@@ -0,0 +1,85 @@
+import { type DataItem, ViewType, type Data, type Route } from '@/types';
+import type { Context } from 'hono';
+import ofetch from '@/utils/ofetch';
+import { load } from 'cheerio';
+import cache from '@/utils/cache';
+import type { Post } from './types';
+import { config } from '@/config';
+import { parseDate } from '@/utils/parse-date';
+
+const WORDPRESS_HASH = 'f05fca638390aed897fbe3c2fff03000';
+
+export const route: Route = {
+    name: '云谦的博客',
```

Comment:
Route name shouldn't be the same as namespace name.

Reply:
thanks, has been fixed